src/components/SpectralProfiler/SpectralProfilerComponent.tsx

```diff
@@ -152,25 +152,40 @@ export class SpectralProfilerComponent extends React.Component<WidgetProps> {
         this.widgetStore.setCursor(x);
     }, 33);
 
-    private genCursoInfoString = (data: Point2D[], smoothedData: Point2D[], cursorXValue: number, cursorXUnit: string, label: string): string => {
-        let cursorInfoString = undefined;
+    private precisionFormatting = (data: number, diff: number, spectralType: SpectralType): string => {
+        return spectralType === SpectralType.CHANNEL ? toFixed(data) : toFormattedNotationByDiff(data, diff);
+    };
+
+    private genCursoInfoString = (data: Point2D[], smoothedData: Point2D[], secondaryXData: number[], cursorXValue: number, cursorXUnit: string, label: string): string => {
```

Comment:
Is this a typo? Is it genCursorInfoString?

Reply:
Eagle eye 🦅

Reply:
good catch! Typo fixed.